apps/vr-tests-react-components/src/stories/MessageBar/MessageBar.stories.tsx

```diff
@@ -64,6 +64,25 @@ export const Multiline = () => (
   </div>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31391&commitId=5744199c2e37061005fb4b2af9aec5345e4637a9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|1| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>MessageBar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Multiline Without Actions.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>